.gitignore

```diff
@@ -30,9 +30,11 @@ fe_plugins/output
 fe/mocked
 fe/ut_ports
 fe/*/target
+fs_brokers/apache_hdfs_broker/src/main/thrift/
```

Comment:
Is this intended?

Reply:
When we compile the fs broker, this directory will generate some temporary files, (some temporary java code generated by thrift) we should not trace them

Reply:
Can you please split this into a different PR?

Reply:
This is a very small change. I don't think it's necessary to separate out another PR

Reply:
As this is a very small change, it should be easy for you to separate it.

Reply:
resolve it